modules/compliance-tailored-profiles.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-tailor.adoc
+
+[id="compliance-tailored-profiles_{context}"]
+= Using tailored profiles
+While the TailoredProfile CR enables the most common tailoring operations, the XCCDF standard allows even more flexibility in tailoring OpenSCAP profiles. In addition, if your organization has been using OpenScap previously, you may have an existing XCCDF tailoring file and can reuse it.
+
+The ComplianceSuite object contains an optional TailoringConfigMap attribute that you can point to a custom tailoring file. The value of the TailoringConfigMap attribute is a name of a ConfigMap which must contain a key called `tailoring.xml` and the value of this key is the tailoring contents.
+
+.Procedure
+
+. Browse the available rules for the rhcos4 ProfileBundle:
++
+[source,terminal]
+----
+oc get rules.compliance \
+-l compliance.openshift.io/profile-bundle=rhcos4
+----
+
+. Browse the available variables in the same ProfileBundle:
++
+[source,terminal]
+----
+oc get variables.compliance \
+-l compliance.openshift.io/profile-bundle=rhcos4
+----
+
+. Pick which rules you want to add to the TailoredProfile. This TailoredProfile example disables two rules and changes one value. Use the `rationale` value to describe why these changes were made:
```

Comment:
s/Pick/Choose

Reply:
` TailoredProfile` 
all other objects and attributes should also be in backticks